common/src/main/java/com/sequenceiq/cloudbreak/telemetry/DataBusEndpointProvider.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.cloudbreak.telemetry;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataBusEndpointProvider {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DataBusEndpointProvider.class);
+
+    private static final String CNAME_ENDPOINT = "https://dbusapi.us-west-1.altus.cloudera.com";
```

Comment:
Are we sure this URL won't change? If not, then maybe it is worth to make a new config property for this in application.yml

Reply:
does not seemed to be changed ever. I can do a follow-up change if that will be needed. although if it changes it should be changed everywhere so actually it sounds more like a constant still

Reply:
ok, then it is fine for me